### DISCUSSION TITLE
Creating jobs without executing.

### DISCUSSION THREAD

**Discussion by wsmckenz** [agenda/agenda#1540] [Q&A]
I have some API code written in typescript/mongoose, and I want to schedule jobs from the API, but not process them in the same execution space. I have a separate standalone container that just processes the jobs. It seems like if I use the 'same' agenda in terms of database/collection, but just dont 'start' it in the API, the sidecar container would see the jobs created by the API and run them? Is this a reasonable approach, or should I create an http endpoint in my sidecar container and have the API code call it to schedule the jobs?

**Comment by reetu40** [comment]
i am also not able to execute the job,I am using js and moongose
